assets/js/app.js

```diff
@@ -113,6 +121,17 @@ window.addEventListener("contextmenu", (event) => {
   }
 });
 
+window.addEventListener("bulk-actions-state", () => {
+  const checkboxes = document.querySelectorAll("[name='session_ids[]']");
+  const disconnect = document.querySelector("[name='disconnect']");
```

Comment:
This is a bit brittle, we should make this more specific like `#edit-sessions [name='disconnect']` (same for the other selectors).